README.rst

```diff
@@ -50,4 +50,4 @@ Install
 Contributing
 ------------
 
-Contributions are welcome, and more information is available in the `contributing guide <CONTRIBUTING.md>`_.
+Contributions are welcome, and more information is available in the `contributing guide <contributing.html>`_.
```

Comment:
does contributing.html work? As in were you able to run `make docs` and see it there

Reply:
I think that link should probably point to what will ultimately be the published page: https://parsenvy.readthedocs.io/en/latest/contributing.html

The current value breaks RTD, and this change breaks GitHub, but linking to the published page should fix both.